packages/mercury/docs/installation-for-angular.md

```diff
@@ -0,0 +1,344 @@
+
+`src/main.ts` <small>(Suggested file)</small>
+
+```ts
+// import { bootstrapApplication } from "@angular/platform-browser";
+// import { appConfig } from "./app/app.config";
+// import { App } from "./app/app";
+
+// Mercury
+import { setBundleMapping } from "@genexus/mercury";
+import { bundleToHashMappings } from "{{ CSS outDir path }}/bundle-to-hash-mappings";
+setBundleMapping(bundleToHashMappings);
+import {
+  registerMercury,
+  getImagePathCallbackDefinitions
+} from "@genexus/mercury";
+registerMercury();
+registryProperty("getImagePathCallback", getImagePathCallbackDefinitions);
+
+// Chameleon
+import { registryProperty } from "@genexus/chameleon-controls-library/dist/collection";
+import { defineCustomElements } from "@genexus/chameleon-controls-library/loader";
+defineCustomElements(window);
+
+//bootstrapApplication(App, appConfig).catch(err => console.error(err));
+```
+
+<details>
+  <summary>Code explained:</summary>
+
+- `setBundleMapping`: Establishes the mapping between bundle names and their generated hashes. For example, it maps the `components/button` bundle name to `button-e261832acea09e81.css`.
+  <br>
+
+- `bundleToHashMappings`: It is an object that holds the mappings between the bundles with the hashed bundles: For example:
+
+```ts
+export const bundleToHashMappings = {
+  "all": "all-5623e5a4aaceb8ac",
+  "base/base": "base/base",
+  "base/base-globant": "base/base-globant-2e35f235ecacc6c3",
+  "base/icons": "base/icons-137edfdbd4d60c97",
+  // and so on ...
+```
+
+- `registerMercury`: (TODO)
+  <br>
+
+- `getImagePathCallbackDefinitions`: An object used to register the `getImagePathCallback` definitions for all controls in Chameleon, which is a property that is present on several Chameleon controls, and it specifies a callback that is executed when the path for an item image needs to be resolved.
+  <br>
+
+- `registryProperty`: (TODO)
+  <br>
+
+- `defineCustomElements`: registers all Chameleon Web Components in the browser’s customElements registry so they can be used in HTML.
+
+</details>
+
+> [!IMPORTANT]
+> Include and run `setBundleMapping` and `bundleToHashMappings` before using any Mercury or [Chameleon](https://github.com/genexuslabs/chameleon-controls-library) utilities!
+
+#### Alternative: Using already generated CSS bundles
+
+> [!WARNING]
+> We don't recommend this way, because the generated CSS contains fixed paths for the assets and the CSS names don't contain a hash, which leads to cache issues when re-deploying your application after updating your version of Mercury.
+> We only include this case, because in some scenarios it can facilitate the initialization with Mercury.
+
+Copy the `<path to node_modules>/@genexus/mercury/dist/bundles/css` content to `{{ CSS outDir path }}` (or the CSS outDir path that you chose).
```

Comment:
This subsection does not go here. It should be another subsection within the second section (Installation).